apps/air-discount-scheme/web/screens/Admin/components/Filters/Filters.tsx

```diff
@@ -176,9 +176,13 @@ function Filters({ onSubmit, defaultValues }: PropTypes) {
                 options={[
                   { value: 'kk', label: 'kk' },
                   { value: 'kvk', label: 'kvk' },
-                  { value: 'hvk', label: 'hvk' },
+                  { value: 'hvk', label: 'x' },
                 ]}
               />
+              <Checkbox
+                name="isExplicit"
+                options={[{ value: 'true', label: 'Handvirkur kóði' }]}
```

Comment:
Thi sshould maybe be internationalised?

Reply:
Looking at this further, turns out the Admin interface isn't internationalised. No part of it.
There is internationalisation in the public facing frontend.

This is a valid comment, however, this Admin interface is soon to be deprecated in favour of the Admin Dashboard being introduced by @eirikurn and others.

I'd rather not go through the work of introducing I18n to this dying interface at this point